### PR TITLE
fix: use paragon branding variables for colors

### PR DESCRIPTION
**Pull request by shahbaz-arbisoft**
We currently have branding issue in our repo. We are hardcoding branding colors in different parts of our code.
Now, we are using paragon branding variables instead of hardcoding the colors.

Tickets: [VAN-1163](https://2u-internal.atlassian.net/browse/VAN-1163)

### CODE REVIEW

File: src/sass/_style.scss [hunk 1]
```diff
@@ -393,7 +393,7 @@ select.form-control {
     line-height: 1.25rem;
     overflow-y: scroll;
     position: absolute;
-    background-color: #fff;
+    background-color: white;
```
Comment (arbrandes) [inline src/sass/_style.scss:396]: Do you mind using `$white` here and elsewhere?

**Review by arbrandes — CHANGES_REQUESTED** [review]
This is awesome, thanks @shahbaz-arbisoft!  I only had one comment.

**Review by arbrandes — APPROVED** [review]
Awesome, thanks!

**Comment by zainab-amir** [comment]
Update: Waiting for UX to approve the color changes.

**Pull request merged**